Source/Scene/Scene.js

```diff
@@ -1805,9 +1861,67 @@ define([
         updatePrimitives(scene);
         createPotentiallyVisibleSet(scene);
         updateAndClearFramebuffers(scene, passState, defaultValue(scene.backgroundColor, Color.BLACK));
-
         executeComputeCommands(scene);
-        executeCommands(scene, passState);
+
+        var viewport = passState.viewport;
+
+        if (scene._useWebVR) {
+            if (frameState.mode !== SceneMode.SCENE2D) {
+                // Based on Calculating Stereo pairs by Paul Bourke
```

Comment:
Put all this in a separate function. https://github.com/AnalyticalGraphicsInc/cesium/blob/master/Documentation/Contributors/CodingGuide/README.md#functions